TCMSApp/src/client/styles/landing-styles.css

```diff
@@ -0,0 +1,82 @@
+ /* Downloaded font */
```

Comment:
yep, it is not less (so you can create the task) :)

less is not a silver bullet, but:
- we use less in main styles file
- I want to control our design efforts better (for example - see all used colors as variables at the bbeginning of our style file)